drools-wb-screens/drools-wb-guided-template-editor/drools-wb-guided-template-editor-client/src/main/java/org/drools/workbench/screens/guided/template/client/editor/TemplateDataCellValueFactory.java

```diff
@@ -314,7 +314,7 @@ public String convertToModelCell(TemplateDataColumn column,
         DataType.DataTypes dataType = getDataType(column);
 
         if (OperatorsOracle.operatorRequiresList(column.getOperator())) {
-            return cv.getValue().toString();
+            return (cv.getValue() == null ? null : cv.getValue().toString());
```

Comment:
@jomarko Ok this null check was missing. I checked return values for other column types and looks like it is normal that the cv.getValue() can be null.